src/OrchardCore/OrchardCore.Users.Core/Services/UserService.cs

```diff
@@ -292,7 +292,9 @@ public void ProcessValidationErrors(IEnumerable<IdentityError> errors, User user
                     case "InvalidEmail":
                         reportError("Email", S["Email '{0}' is invalid.", user.Email]);
                         break;
-
+                    case "InvalidToken":
+                        reportError(nameof(user.ResetToken), S["Token '{0}' is invalid.", user.ResetToken]);
```

Comment:
Not sure if it's a good or bad idea to output the token itself. I know it's not sensitive info, but it is usually a long string of gibberish.

Regardless, this change at least provides an easier way to customize the text with a PO file, so that is a welcome change.

Reply:
Agree, no need to show the reset token